Makefile

```diff
@@ -0,0 +1,241 @@
+# /!\ /!\ /!\ /!\ /!\ /!\ /!\ DISCLAIMER /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\
+#
+# This Makefile is only meant to be used for DEVELOPMENT purpose as we are
+# changing the user id that will run in the container.
+#
+# PLEASE DO NOT USE IT FOR YOUR CI/PRODUCTION/WHATEVER...
+#
+# /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\
+#
+# Note to developers:
+#
+# While editing this file, please respect the following statements:
+#
+# 1. Every variable should be defined in the ad hoc VARIABLES section with a
+#    relevant subsection
+# 2. Every new rule should be defined in the ad hoc RULES section with a
+#    relevant subsection depending on the targeted service
+# 3. Rules should be sorted alphabetically within their section
+# 4. When a rule has multiple dependencies, you should:
+#    - duplicate the rule name to add the help string (if required)
+#    - write one dependency per line to increase readability and diffs
+# 5. .PHONY rule statement should be written after the corresponding rule
+# ==============================================================================
+# VARIABLES
+
+BOLD := \033[1m
+RESET := \033[0m
+GREEN := \033[1;32m
+
+# -- Docker
+# Get the current user ID to use for docker run and docker exec commands
+DOCKER_UID          = $(shell id -u)
+DOCKER_GID          = $(shell id -g)
+DOCKER_USER         = $(DOCKER_UID):$(DOCKER_GID)
+COMPOSE             = DOCKER_USER=$(DOCKER_USER) docker compose
+COMPOSE_EXEC        = $(COMPOSE) exec
+COMPOSE_RUN         = $(COMPOSE) run --rm --build
+
+
+# ==============================================================================
+# RULES
+
+default: help
+
+# -- Project
+
+create-env-files: ## Create empty .local env files for local development
+create-env-files: \
+	ops/env/widgets.local \
+	ops/env/website.local
+.PHONY: create-env-files
+
+bootstrap: ## Prepare the project for local development
```

Comment:
Petit retour : le texte _Prepare the project […]_ ici, qui est donc affiché quand on lance `make help`, mérite peut-être qu'on soit plus clair que ça lance vraiment les services ? Quand je lis "prepare" j'avoue avoir tendance à comprendre que ça va juste me build les trucs nécessaires après avoir cloné une 1ère fois. Alors que ça lance le serveur local en prime.